### ISSUE TITLE
At some point lightnode stops loading blocks

### ISSUE THREAD

**Issue by vldm** [stegos/stegos#1485]
```
2020-02-11 13:25:08 INFO [stegos_wallet::storage] Applied a micro block: epoch=12295, offset=58, block=69d21da81ee8
2020-02-11 13:25:17 INFO [stegos_wallet::storage] Applied a micro block: epoch=12295, offset=59, block=e83f6ff94435
```

last messages is about microblock, and then light node stops loading.

**Comment by rtsisyk** [comment]
It can stop to load in case of rollback/forkresolution. There is nothing else to do except restarting the replication by choosing a new upstream. This procedure is performed automatically by timer. I will improve the implementation to restart in case of error too.

**Comment by rtsisyk** [comment]
Fixed in #1439
